tests/unittests/core/cli/test_info.py

```diff
@@ -0,0 +1,949 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Collection of tests for :mod:`orion.core.cli.info`."""
+import itertools
+
+import pytest
+
+from orion.core.cli.info import (
+    format_algorithm, format_commandline, format_config, format_dict, format_info, format_list,
+    format_metadata, format_refers, format_space, format_stats, format_title, get_trial_params)
+from orion.core.io.space_builder import SpaceBuilder
+from orion.core.worker.trial import Trial
+
+
+class DummyExperiment():
+    """Dummy container to mock experiments"""
+
+    pass
+
+
+@pytest.fixture
+def dummy_trial():
+    """Return a dummy trial object"""
+    trial = Trial()
+    trial.params = [
+        Trial.Param(name='a', type='real', value=0.0),
+        Trial.Param(name='b', type='integer', value=1),
+        Trial.Param(name='c', type='categorical', value='Some')]
+    return trial
+
+
+@pytest.fixture
+def dummy_dict():
+    """Return a dict of dicts"""
+    return {
+        1: {
+            1.1: "1.1.1",
+            1.2: {
+                "1.2.1": {},
+                "1.2.2": "1.2.2.1"
+            }
+        },
+        2: {
+            2.1: "2.1.1",
+            2.2: {}
+        },
+        3: {}
+    }
+
+
+@pytest.fixture
+def dummy_list_of_lists():
+    """Return an list of lists"""
+    return [
+        1,
+        [2, 3],
+        4,
+        [5, 6, 7, 8]
+    ]
+
+
+@pytest.fixture
+def dummy_list_of_objects(dummy_dict):
+    """Return an list of objects"""
+    return [
+        {
+            1: {
+                1.1: "1.1.1",
+                1.2: [
+                    "1.2.1",
+                    "1.2.2"
+                ]
+            },
+        },
+        [
+            4, 5
+        ],
+        {
+            2: {
+                2.1: "2.1.1",
+                2.2: {}
+            },
+            3: {}
+        }
+    ]
+
+
+@pytest.fixture
+def algorithm_dict():
+    """Return an algorithm configuration"""
+    return dict(
+        bayesianoptimizer=dict(
+            acq_func='gp_hedge',
+            alpha=1e-10,
+            n_initial_points=10,
+            n_restarts_optimizer=0,
+            normalize_y=False,
+            strategy='cl_min'))
```

Comment:
We don't support `strategy` anymore, should we remove this?

Reply:
Indeed!